packages/block-library/src/post-comments/index.php

```diff
@@ -37,8 +35,15 @@ function render_block_core_post_comments( $attributes, $content, $block ) {
 	}
 
 	$wrapper_attributes = get_block_wrapper_attributes( array( 'class' => $classes ) );
-	$output             = sprintf( '<div %1$s>', $wrapper_attributes ) . ob_get_clean() . '</div>';
-	return $output;
+	$output             = ob_get_clean();
+
+	// There's a deprecation warning generated by WP Core.
+	// Ideally this deprecation is removed from Core.
+	// In the meantime, this removes it from the output.
+	$regex  = '/Deprecated:((.)*\n){3}/m';
```

Comment:
Is this deprecation warning always printed in English?

Reply:
Maybe you could narrow it down to matching `comments.php` and `wp-includes/functions.php`?